src/plays/Quote Generator/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```

Comment:
Do you need this file?

Reply:
> @Kumar-Ankit56 how did you create this play? Have you used the create play command to build the initial play structure?
> 
> I do not see any of the play file structure also the header and other components are missing.
> 
> If you follow the commands in createplay md file then it will create basic project structure for you and you just need to drop your business logic into it.
> 
> Please let me know if you need help.

Sir haven't use play structure but let me try. If will not able to do.
Otherwise I will ping you sir.

Reply:
Yes plz. Without that the build will not pass.